--- apache2.conf.orig	2011-01-31 07:06:34.000000000 -0500
+++ /etc/apache2/apache2.conf	2011-01-31 07:13:37.000000000 -0500
@@ -86,8 +86,8 @@
 #
 # KeepAliveTimeout: Number of seconds to wait for the next request from the
 # same client on the same connection.
-#
-KeepAliveTimeout 15
+
+KeepAliveTimeout 1
 
 ##
 ## Server-Pool Size Regulation (MPM specific)
@@ -100,11 +100,11 @@
 # MaxClients: maximum number of server processes allowed to start
 # MaxRequestsPerChild: maximum number of requests a server process serves
 <IfModule mpm_prefork_module>
-    StartServers          5
+    StartServers          20
     MinSpareServers       5
-    MaxSpareServers      10
+    MaxSpareServers      15
     MaxClients          150
-    MaxRequestsPerChild   0
+    MaxRequestsPerChild   10000
 </IfModule>
 
 # worker MPM

