--- Config.pm.orig	2010-08-17 15:40:10.000000000 -0400
+++ Config.pm	2010-08-17 16:25:16.000000000 -0400
@@ -51,7 +51,7 @@
   'perl5lib_verbosity' => q[v],
   'prefer_installer' => q[MB],
   'prefs_dir' => q[/root/.cpan/prefs],
-  'prerequisites_policy' => q[ask],
+  'prerequisites_policy' => q[follow],
   'scan_cache' => q[atstart],
   'shell' => q[/bin/bash],
   'show_unparsable_versions' => q[0],
