--- /etc/ejabberd/ejabberd.cfg	2009-09-04 09:00:04.000000000 -0400
+++ ejabberd.lenny.cfg	2009-09-04 08:58:05.000000000 -0400
@@ -33,7 +33,7 @@
 {acl, admin, {user, "", "localhost"}}.
 
 %% Hostname
-{hosts, ["localhost"]}.
+{hosts, ["public.localhost", "private.localhost"]}.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
@@ -115,7 +115,7 @@
   {5222, ejabberd_c2s, [
 			{access, c2s},
 			{shaper, c2s_shaper},
-			{max_stanza_size, 65536},
+			{max_stanza_size, 200000},
 			starttls, {certfile, "/etc/ejabberd/ejabberd.pem"}
 		       ]},
 
@@ -125,13 +125,13 @@
   %%{5223, ejabberd_c2s, [
   %%			{access, c2s},
   %%			{shaper, c2s_shaper},
-  %%			{max_stanza_size, 65536},
+  %%			{max_stanza_size, 200000},
   %%			tls, {certfile, "/etc/ejabberd/ejabberd.pem"}
   %%		       ]},
 
   {5269, ejabberd_s2s_in, [
 			   {shaper, s2s_shaper},
-			   {max_stanza_size, 131072}
+			   {max_stanza_size, 200000}
 			  ]},
 
   %% External MUC jabber-muc (but internal mod_muc is better :))
@@ -350,12 +350,12 @@
 %%
 %% The "normal" shaper limits traffic speed to 1.000 B/s
 %%
-{shaper, normal, {maxrate, 1000}}.
+{shaper, normal, {maxrate, 500000}}.
 
 %%
 %% The "fast" shaper limits traffic speed to 50.000 B/s
 %%
-{shaper, fast, {maxrate, 50000}}.
+{shaper, fast, {maxrate, 500000}}.
 
 
 %%%   ====================
@@ -392,7 +392,7 @@
 %%%   ACCESS RULES
 
 %% Define the maximum number of time a single user is allowed to connect:
-{access, max_user_sessions, [{10, all}]}.
+{access, max_user_sessions, [{5000, all}]}.
 
 %% This rule allows access only for local users:
 {access, local, [{allow, local}]}.
@@ -466,7 +466,7 @@
 		  {max_users, 500}
 		 ]},
   %%{mod_muc_log,[]},
-  {mod_offline,  []},
+  %%{mod_offline,  []},
   {mod_privacy,  []},
   {mod_private,  []},
   {mod_proxy65,  [
