--- opensrf_core.xml.example	2011-06-18 11:32:06.000000000 -0400
+++ opensrf_core.xml.example.patched	2011-08-08 14:25:21.000000000 -0400
@@ -51,7 +51,7 @@
     <!-- Our domain should match that of the private router -->
     <domain>private.localhost</domain>
     <username>opensrf</username>
-    <passwd>password</passwd>
+    <passwd>OpenSRF_Password</passwd>
     <port>5222</port>
 
     <!-- 
@@ -95,7 +95,7 @@
 
     <!-- jabber login info -->
     <username>opensrf</username>
-    <passwd>password</passwd>
+    <passwd>OpenSRF_Password</passwd>
     <port>5222</port>
     <loglevel>3</loglevel>
     <logfile>LOCALSTATEDIR/log/gateway.log</logfile>
@@ -129,7 +129,7 @@
         <port>5222</port>
         <unixpath>LOCALSTATEDIR/sock/unix_sock</unixpath>
         <username>router</username>
-        <password>password</password>
+        <password>OpenSRF_Password</password>
         <resource>router</resource>
         <connect_timeout>10</connect_timeout>
         <max_reconnect_attempts>5</max_reconnect_attempts>
@@ -154,7 +154,7 @@
         <server>private.localhost</server>
         <port>5222</port>
         <username>router</username>
-        <password>password</password>
+        <password>OpenSRF_Password</password>
         <resource>router</resource>
         <connect_timeout>10</connect_timeout>
         <max_reconnect_attempts>5</max_reconnect_attempts>
